azure-pipelines.yml

```
@@ -0,0 +1,40 @@
+# Docker
+# Build and push an image to Azure Container Registry
+# https://docs.microsoft.com/azure/devops/pipelines/languages/docker
+
+trigger:
+- develop
```

We want to have this also on `master`, which is where we would usually deploy from, although I guess the the automated deployment from `master` should be re-established

```suggestion
- master
- develop
```